instana-java-sdk/src/main/java/com/instana/sdk/support/SpanSupport.java

```diff
@@ -6,21 +6,26 @@
 import com.instana.sdk.annotation.Span;
 
 /**
- * This support class allows for recording additional information when recording a {@link Span}.
+ * This support class allows for specifying additional information when recording a {@link Span}.
+ * <p>
+ * Most methods work on spans created by the SDK and spans created by the automatic trace instrumentation,
+ * known as Instana AutoTrace. Details on which kind of spans can be created or manipulated are explained
```

Comment:
...as Instana AutoTrace. Some limitations may apply due to technical limitations in case of asynchronous frameworks and libraries. Details on...

Reply:
:+1: